src/locale/cs_CZ.po

```diff
@@ -31,11 +31,17 @@ msgid "Alert background color"
 msgstr "Barva pozadí výstrahy"
 
 #: src/prefs.ui:75 src/prefs.ui:79
-msgid "Alert even if you set notifications to OFF on user menu (default: OFF)"
+#, fuzzy
+msgid "Alert even if you enable 'Do Not Disturb' (default: OFF)"
```

Comment:
I prefer to revert the changes to those translations if they are not going to be updated